packages/web/env-config.js.template

```diff
@@ -8,7 +8,7 @@ const environments = {
       databaseURL: 'https://celo-faucet.firebaseio.com',
       projectId: 'celo-faucet',
     },
-    LEADERBOARD: {
+    BLOCKSCOUT: {
```

Comment:
this is only the template, if the secrets file isnt changed this wont matter

Reply:
I don't know how or who can change the file. Will ask tomorrow

Reply:
ok maybe we do the leaderboard to blockscout change in a separate PR ?

Reply:
looks like connor handled it